src/settings/AcquisitionUnits/AcquisitionUnitsList/AcquisitionUnitsList.js

```diff
@@ -30,10 +37,11 @@ const AcquisitionUnitsList = ({ acquisitionUnits, getViewPath, getCreatePath })
       paneTitle={<FormattedMessage id="ui-acquisition-units.meta.title" />}
     >
       <NavList data-test-ac-units-nav-list>
-        {acquisitionUnits.map(acquisitionUnit => (
+        {acquisitionUnits.map((acquisitionUnit, i) => (
           <NavListItem
             key={acquisitionUnit.id}
-            to={getViewPath(acquisitionUnit.id)}
```

Comment:
what is the reason of changing `to` to `onClick` ?

Reply:
By using the onClick-prop for <NavListItem> it will render a button that could be autofocused